src/api/handler/handler.js

```diff
@@ -1,17 +1,25 @@
-import {axiosNasaClient} from "../../clients/index.js";
+import {nasaAsteroidService} from "../../services/index.js";
 import {getPreviousPeriod, parseDate} from "./dateUtils.js";
 
+const parseQueryParams = (query) => {
+    return {
+        date: query.date,
+        countOnly: Boolean(query.count),
+        dangerousOnly: Boolean(query["were-dangerous-meteors"]),
```

Comment:
Please follow the same way to retrieve the `query` properties - dot syntax or array access

Reply:
Fixed.